content/scripts/build.js

```diff
@@ -527,7 +534,7 @@ class Builder {
 
     fs.writeFileSync(
       allTitlesJsonFilepath,
-      JSON.stringify(this.allTitles, null, 2)
+      JSON.stringify(mapToObject(this.allTitles), null, 2)
```

Comment:
Since all of the keys in `this.allTitles` are strings, I don't see why you have to convert the `Map` to an object, since the `JSON.stringify` works just fine on `Map` objects.

Reply:
```javascript
const allTitles = new Map(Object.entries({ foo: "Foo", bar: "Bar" }));
console.log(allTitles)
console.log('---');
console.log(JSON.stringify(allTitles))
console.log(JSON.stringify(...allTitles))
console.log(JSON.stringify({...allTitles}))
```
outputs:
```
Map(2) { 'foo' => 'Foo', 'bar' => 'Bar' }
---
{}
["foo","Foo"]
{}
```

Reply:
Wow, now I'm really confused:
```javascript
> x = new Map([['x', 1], ['y', 2]]);
Map { 'x' => 1, 'y' => 2 }
> x
Map { 'x' => 1, 'y' => 2 }
> JSON.stringify(x);
'{}'
> x = new Map();
Map {}
> x['x'] = 1;
1
> x['y'] = 2;
2
> x
Map { x: 1, y: 2 }
> JSON.stringify(x);
'{"x":1,"y":2}'
>
```
I wonder if in the first case the `Map` constructor places the keys and values in the prototype, which is why its `JSON.stringify` is empty, while in the second case (the one I tried originally when making my review comment), the keys and values are actually stored as the object's "own" properties?

Reply:
Ah, yes, that's the case:
```javascript
> x.hasOwnProperty('x')
true
> x = new Map([['x', 1], ['y', 2]]);
Map { 'x' => 1, 'y' => 2 }
> x.hasOwnProperty('x')
false
>
```

Reply:
Oh wait, what an idiot I am! In the examples above, I'm not using the `Map` methods. 🤦 
```javavscript
> x = new Map();
Map {}
> x.set('x', 1);
Map { 'x' => 1 }
> x.hasOwnProperty('x')
false
>
```

Reply:
["Beware!" they say](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Usage_notes)